app.js

```diff
@@ -1,10 +1,10 @@
-const { application } = require("express");
+const app  = express()
```

Comment:
You can't use `express()` before requiring it, you need to require express first and then use it:

```
const express = require("express");
const app  = express()
```